app/controllers/content_item_signups_controller.rb

```diff
@@ -59,6 +55,18 @@ def content_item
       .content_item(content_item_path)
   end
 
+  def handle_redirects
+    if content_item["document_type"] == "redirect"
+      destination_path = content_item.dig("redirects", 0, "destination")
```

Comment:
Does this mean we added something to the document_type too then?

The previous one redirected to based on the topic?